geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/SimpleFeatureTypes.scala

```diff
@@ -53,7 +53,6 @@ object SimpleFeatureTypes {
     val OverrideReservedWords = "override.reserved.words"
     val QueryInterceptors     = "geomesa.query.interceptors"
     val StatsEnabled          = "geomesa.stats.enable"
-    val TableCompression      = "geomesa.table.compression.enabled"
```

Comment:
Should we note in the upgrade guide that this configuration is gone?

Reply:
we still support the old functionality, so i think it's ok to not mention it. if we ever drop that support we can mention it.